src/compaction_execute_bench.rs

```diff
@@ -64,7 +64,9 @@ fn open_object_store(options: &Options) -> Result<Arc<dyn ObjectStore>, SlateDBE
 pub fn run_compaction_execute_bench() -> Result<(), SlateDBError> {
     let options = load_options();
     let s3 = open_object_store(&options)?;
-    let sst_format = SsTableFormat::new(4096, 1, options.compression_codec);
+    let sst_format = SsTableFormatBuilder::new()
```

Comment:
I like this! I do wonder: what's idiomatic in Rust? I could see an argument for:

```rust
SsTableFormat::new(
  compression_codec: options.compression_codec,
  ..Default::default(),
);
```

Reply:
yep, this is is the right way to do this in rust I think. I'll change it.